src/evm/onchain/onchain.rs

```diff
@@ -313,6 +314,10 @@ where
                                  .collect::<Vec<_>>(),
                              caller
                     );
+
+                    if !host.code.contains_key(&caller) {
```

Comment:
why do you need to make caller have code as the target?

Reply:
Here, the caller is also a proxy contract. It usually appears in cases where bytecode is used directly within the contract or when a temporary contract is created using "create". In the case of newly created temporary contracts, when the "execute_from_pc" function executes a specific function, bytecode is required to meet the conditions for continued execution.